### PR TITLE
nick11967 10026 judge

### PR DESCRIPTION
**Pull request by nick11967**

### CODE REVIEW

**Comment by nick11967** [comment]
image
What is '---' ?

File: nick11967/Ch 4/29 Shoemakers Problem/shoemakers_problem.cpp [hunk 1]
```diff
@@ -2,15 +2,13 @@
...
-#define S second
-
 int main()
 {
     int c;
     scanf("%d", &c);
     int n;
-    vector<pair<int, int>> work;
+    vector<tuple<int, int, int>> work;
+    tuple<int, int, int> temp;
```
Comment (gracekim027) [inline nick11967/Ch 4/29 Shoemakers Problem/shoemakers_problem.cpp:11]: 제거

File: nick11967/Ch 4/29 Shoemakers Problem/shoemakers_problem.cpp [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
-#define T first
-#define S second
-
 int main()
 {
     int c;
     scanf("%d", &c);
     int n;
-    vector<pair<int, int>> work;
+    vector<tuple<int, int, int>> work;
```
Comment (gracekim027) [inline nick11967/Ch 4/29 Shoemakers Problem/shoemakers_problem.cpp:10]: Pair 사용

Comment (nick11967) [reply nick11967/Ch 4/29 Shoemakers Problem/shoemakers_problem.cpp:10]: pair<pair<int,int>,int>

**Comment by gracekim027** [comment]
LGTM!

**Pull request merged**